src/Extension/Settings/settings.yaml

```diff
@@ -1,4 +1,18 @@
 settings:
+  - path: developer_email
```

Comment:
Seems like this changes are from another epic.

Reply:
removed